app/scripts/lib/app-start.js

```diff
@@ -110,6 +111,7 @@ function (
   User,
   FormPrefill,
   Notifications,
+  PageRefreshWatch,
```

Comment:
https://github.com/mozilla/fxa-content-server/issues/3185

Page in this situation is CMD-R refresh. `BrowserRefreshWatch`?